pkg/integrations/v4/supervisor_fb.go

```diff
@@ -98,6 +107,17 @@ func buildFbExecutor(fbIntCfg FBSupervisorConfig, cfgLoader *logs.CfgLoader) fun
 			return nil, errors.Wrap(err, "failed to create temporary fb sFBLogger config file")
 		}
 
+		if removedFbConfigTempFiles, err := removeFbConfigTempFiles(MaxNumberOfFbConfigTempFiles); err != nil || removedFbConfigTempFiles != nil {
```

Comment:
I would distinguish between the error and not error scenarios for readability. Also being `removedFbConfigTempFiles` a slice it can be iterated w/o checking 
```go
		removedFbConfigTempFiles, err := removeFbConfigTempFiles(MaxNumberOfFbConfigTempFiles)
		if err != nil {
			log.WithError(err).Warn("Failed removing config temp files.")
		}

		for _, file := range removedFbConfigTempFiles {
			log.Debugf("Removed %s config temp file.", file)
		}
```